sample.env:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]

```

Comment:
I'm not sure - these zone identifier files populate occasionally in my WSL repo. I've just been deleting them since I don't think they should be there. A cursory search claims that they're simply metadata about a file's origin (like when I re-downloaded sample.env back into the repo):

"Zone.Identifier files are a Windows-specific feature that stores metadata about a file’s origin, typically when files are downloaded from the internet. These files are often hidden but may appear in repositories when inadvertently added or copied. Here’s why they show up and how to address them"